### ISSUE TITLE
Handle "name too different" errors better

### ISSUE THREAD

**Issue by tmtmtmtm** [mysociety/verification-pages#176]
When verifying the data for Rammohan Naidu Kinjarapu, I tried to say that the name at the given URL is slightly different ("Ram Mohan Naidu,Shri Kinjarapu"), but I got a pop-up error saying that that's *too* different, with no indication of what I'm meant to do about it, or why that's even a problem:

![screen shot 2018-06-28 at 11 35 41](https://user-images.githubusercontent.com/57483/42029512-b7bfe09e-7ac7-11e8-8c4f-fc99fd58702c.png)

**Comment by tmtmtmtm** [comment]
I think this is no longer relevant after #355
